eZ/Publish/Core/Persistence/Legacy/Tests/Content/StorageHandlerTest.php

```diff
@@ -127,6 +128,34 @@ public function testGetFieldDataNotAvailable()
             ->with($this->equalTo('foobar'))
             ->will($this->returnValue($storageMock));
 
+        $field = new Field();
+        $field->id = 123;
+        $field->type = 'foobar';
+        $field->value = new FieldValue();
+
+        $handler = $this->getStorageHandler();
+        $handler->getFieldData($this->getVersionInfoMock(), $field);
+    }
+
+    /**
+     * @covers \eZ\Publish\Core\Persistence\Legacy\Content\StorageHandler::getFieldData
+     */
```

Comment:
Using phpdoc `@covers` annotation on a method is highly discouraged by PHPUnit.

The effect of this annotation is that it limits coverage report (which we don't really use) to this method only, while there is potential for private/protected methods to be called too.

I'd remove it.

```suggestion
```